client/signup/steps/woocommerce-install/index.tsx

```diff
@@ -20,6 +23,27 @@ interface WooCommerceInstallProps {
 export default function WooCommerceInstall( props: WooCommerceInstallProps ): ReactElement | null {
 	const { __ } = useI18n();
 
+	const {
+		stepName,
+		isReskinned,
+		goToStep,
+		stepSectionName,
+		headerTitle,
+		headerDescription,
+	} = props;
+
+	if ( stepName === 'confirm' ) {
+		return (
+			<Confirm
+				goToStep={ goToStep }
+				isReskinned={ isReskinned }
+				stepSectionName={ stepSectionName }
+				headerTitle={ headerTitle }
+				headerDescription={ headerDescription }
+			/>
+		);
+	}
```

Comment:
If you want, we can avoid this with a patch to `step-components.js`

```
diff --git a/client/signup/config/step-components.js b/client/signup/config/step-components.js
index deba56c63e..cc8332711a 100644
--- a/client/signup/config/step-components.js
+++ b/client/signup/config/step-components.js
@@ -81,7 +81,7 @@ const stepNameToModuleName = {
        list: 'import',
        capture: 'import',
        ready: 'import',
-       confirm: 'woocommerce-install',
+       confirm: 'woocommerce-install/confirm',
        transfer: 'woocommerce-install',
        install: 'woocommerce-install',
        complete: 'woocommerce-install',
@@ -94,7 +94,7 @@ export function getStepModuleName( stepName ) {
 export async function getStepComponent( stepName ) {
        const moduleName = stepNameToModuleName[ stepName ];
        const module = await import(
-               /* webpackChunkName: "async-load-signup-steps-[request]", webpackInclude: /signup\/steps\/[0-9a-z-]+\/index.[j|t]sx$/ */ `calypso/signup/steps/${ moduleName }`
+               /* webpackChunkName: "async-load-signup-steps-[request]", webpackInclude: /signup\/steps\/[0-9a-z/-]+\/index.[j|t]sx$/ */ `calypso/signup/steps/${ moduleName }`
        );
        return module.default;
 }
```

I think I'll do this for the transfer step once this is merged.

Reply:
Doing this here https://github.com/Automattic/wp-calypso/pull/58333